indico/modules/events/persons/templates/email_dialog.html

```diff
@@ -0,0 +1,19 @@
+{% from 'forms/_form.html' import simple_form %}
+{{ simple_form(form, submit=_('Send')) }}
+<script>
+    (function() {
+        'use strict';
+
+        var $recipients = $('#form-group-recipients').find('.form-field');
+        var count = {{ emails_count }};
```

Comment:
So, when using jinja interpolated in JS we first assign it to a variable so that syntax highlight doesn't look as broken. Is that the [guideline](https://github.com/indico/indico/wiki/Jinja-guidelines)?